sql/parser/builtins.go

```diff
@@ -399,15 +401,46 @@ var builtins = map[string][]builtin{
 		builtin{
 			types:      typeList{},
 			returnType: DummyDate,
+			fn: func(e EvalContext, args DTuple) (Datum, error) {
+				return DDate{Time: e.StmtTimestamp.Truncate(24 * time.Hour)}, nil
+			},
+		},
+	},
+
+	"statement_timestamp": {nowImpl},
+	"current_timestamp":   {nowImpl},
+	"now":                 {nowImpl},
```

Comment:
Yes, they all share `e.StmtTimestamp`, but they all independently choose to use StmtTimestamp instead of TxnTimestamp. What I'm saying is that all of these should call into one place so that if we decide to make `now()` synonymous with `transaction_timestamp` there is only one place to change.